label_studio/tests/tasks_api.tavern.yml

```diff
@@ -24,4 +24,161 @@ stages:
     method: GET
     url: '{django_live_url}/api/tasks/{task_pk}?project={project_pk}'
   response:
-    status_code: 200
\ No newline at end of file
+    status_code: 200
+
```

Comment:
For data manager filters we have a special folder:
![image](https://user-images.githubusercontent.com/501780/208494186-da178b94-de44-4e91-b5f9-cd07d3acff95.png)